asset_dashboard/templates/asset_dashboard/partials/forms/add_edit_phase_form.html

```diff
@@ -162,6 +185,38 @@ <h3>Phase Assets</h3>
   <script type="text/javascript">
     window.props = JSON.parse(document.getElementById('props').textContent)
     window.reactMount = document.getElementById('map')
+    // Don't worry about auto-saving or funding stream forms if it's a new phase
+    if (!`{{ phase }}`) {
+      throw new Error("This is a new phase and will need to be saved manually")
+    }
+
+    $(".icon-loader, .icon-failed").hide()
+
+    let delayTimer
+    function trackChanges() {
+      // All visible inputs will trigger a save if any they detect any changes
+      const allInputs = $(":input:not(.btn.btn-info, :hidden)")
+      allInputs.on("change keyup", function(e) {
+        // Ignore left and right arrow key inputs
+        if (e.keyCode == 37 || e.keyCode == 39){
+          return
+        }
```

Comment:
The up and down arrows can signify a change in the phase year input, but left and right don't do anything anywhere